with-aws-storage-upload/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "main": "index.js",
```

Comment:
This file doesn't exist

Reply:
Thanks for the review.  
I have updated the package.json